miner/scroll_worker.go

```diff
@@ -517,7 +517,11 @@ func (w *worker) startNewPipeline(timestamp int64) {
 	}
 
 	w.currentPipelineStart = time.Now()
-	w.currentPipeline = pipeline.NewPipeline(w.chain, w.chain.GetVMConfig(), parentState, header, nextL1MsgIndex, w.getCCC()).WithBeforeTxHook(w.beforeTxHook)
+	pipelineCCC := w.getCCC()
+	if !w.isRunning() {
```

Comment:
`isRunning` will only be true on `signer-0` and `signer-1`, correct? And follower nodes (like `internal-1`) will run ccc in the block validation part.

Reply:
`isRunning` will only be true for nodes with `--mine` flag, so whichever signer that is active at that time. This is how the original worker behaved as well. `--ccc` flag doesn't matter in block building.

> And follower nodes (like internal-1) will run ccc in the block validation part.

Yeap, CCC in validation step is configured by the `--ccc` flag as it was before.

Reply:
Fyi both signers have the `--mine` flag, it's just that one of them is not an authorized Clique signer so it will fail at the sealing step. (There might be transient periods where both are active, when we're "rotating" signers.)

Reply:
Hmm, I wasn't aware. That might be an issue. Let me see.

Reply:
But it is unrelated to this PR, so we can merge this when we can.